src/pages/services-jeunes/index.page.test.tsx

```diff
@@ -131,83 +140,220 @@ describe('Page Services Jeunes', () => {
+								<ServicesJeunePage serviceJeuneList={serviceJeuneList} />
+							</DependenciesProvider>,
+						);
+
+						// Then
+						const mesuresJeunesSection = screen.getByRole('region', { name: 'les services jeunes' });
+						const servicesJeunesList = within(mesuresJeunesSection).getAllByRole('list')[1];
+						const servicesJeunesEtiquettes = within(servicesJeunesList).getAllByRole('listitem');
+						expect(servicesJeunesEtiquettes.length).toBe(3);
+					});
+				});
 			});
-			it('affiche un bouton voir plus quand il y a plus de 6 services', () => {
+			describe('Liste de résultats', () => {
+				it('affiche un message d’erreur quand aucun service n’est disponible', () => {
+					// Given
+					mockUseRouter({ query: { filtre : ['Accompagnement'] } });
+					const serviceJeuneList = [
+						aServiceJeune({ categorie: ServiceJeune.Categorie.AIDES_FINANCIERES, titre: 'service 2' }),
+					];
+					const analyticsService = aManualAnalyticsService();
+
+					// When
+					render(
+						<DependenciesProvider analyticsService={analyticsService}>
+							<ServicesJeunePage serviceJeuneList={serviceJeuneList} />
+						</DependenciesProvider>,
+					);
+
+					// Then
+					const mesuresJeunesSection = screen.getByRole('region', { name: 'les services jeunes' });
+					const messageErreur = within(mesuresJeunesSection).getByText('Aucun service jeune disponible. Veuillez modifier votre sélection.');
+					expect(messageErreur).toBeVisible();
+				});
+				it('affiche au maximum 6 services initialement', () => {
 				// Given
-				const serviceJeuneList = [
-					aServiceJeune({ titre: 'service 1' }),
-					aServiceJeune({ titre: 'service 2' }),
-					aServiceJeune({ titre: 'service 3' }),
-					aServiceJeune({ titre: 'service 4' }),
-					aServiceJeune({ titre: 'service 5' }),
-					aServiceJeune({ titre: 'service 6' }),
-					aServiceJeune({ titre: 'service 7' }),
-				];
-				const analyticsService = aManualAnalyticsService();
-
-				// When
-				render(
-					<DependenciesProvider analyticsService={analyticsService}>
-						<ServicesJeunePage serviceJeuneList={serviceJeuneList} />
-					</DependenciesProvider>,
-				);
-
-				// Then
-				const mesuresJeunesSection = screen.getByRole('region', { name: 'les services jeunes' });
-				const voirPlusDeServicesJeunesBouton = within(mesuresJeunesSection).getByRole('button', { name: 'Voir plus de services conçus pour les jeunes' });
-				expect(voirPlusDeServicesJeunesBouton).toBeVisible();
-			});
-			it('affiche un bouton voir moins quand plus de 6 services jeunes sont visibles', async () => {
+					const serviceJeuneList = [
+						aServiceJeune({ titre: 'service 1' }),
+						aServiceJeune({ titre: 'service 2' }),
+						aServiceJeune({ titre: 'service 3' }),
+						aServiceJeune({ titre: 'service 4' }),
+						aServiceJeune({ titre: 'service 5' }),
+						aServiceJeune({ titre: 'service 6' }),
+						aServiceJeune({ titre: 'service 7' }),
+					];
```

Comment:
```suggestion
					const serviceJeuneList = new Array(7).fill(aServiceJeune());
```
À tester, pas certain que ça fonctionne, mais comme ça c'est plus explicite que tu en met 7, c'est écrit en tout chiffre